pdc/apps/release/serializers.py

```diff
@@ -91,12 +92,16 @@ class ReleaseSerializer(StrictSerializerMixin, serializers.ModelSerializer):
                                           allow_null=True,
                                           default=None)
     allow_buildroot_push = serializers.BooleanField(default=False)
+    allowed_debuginfos = ChoiceSlugField(slug_field='name',
+                                         many=True, queryset=Service.objects.all(),
+                                         allow_null=True, required=False, default=[])
```

Comment:
Shouldn't this be `allow_null=False`?

Reply:
Actually it's not , this value can be set as null ([]).

Reply:
`allow_null` just means that you can pass `None` in serializer ([docs](http://www.django-rest-framework.org/api-guide/fields/#allow_null)). Setting value to `null` is disallowed in model (`[]` is not `null`).

Would anything break if you remove it?

Reply:
Aha, thanks for the explain, remove this, and just copy from the ReleaseGroupSerializer.releases, they are similar.

Moved.